qcodes/tests/test_wrong_address.py

```diff
@@ -12,15 +12,15 @@
 
 
 def test_wrong_address():
-    wrong_address = 'GPIB::2::INSTR'
+    wrong_address = 'GPIB0::2::0::INSTR'
```

Comment:
Is the secondary address (the last 0) really needed ? I suspect if that is also true for real instruments that could break a lot of experiments.

Reply:
This is the normalized form of the address and both forms should behave in an identical manner for all implementation of VISA.